lua/nvim-paredit/init.lua

```diff
@@ -8,31 +8,50 @@ local M = {
   api = require("nvim-paredit.api"),
 }
 
-function M.setup(opts)
-  config.update_config(common.merge(defaults.defaults, opts))
+local function setup_keybingings(filetype, buf)
+  local filetypes = config.config.filetypes
+  local keys = config.config.keys
+
+  if common.included_in_table(filetypes, filetype) then
+    keybindings.setup_keybindings({
+      keys = keys,
+      buf = buf,
```

Comment:
This should be `buf = buf or 0` or we should pass in `0` on line 48

Reply:
done